### ISSUE TITLE
Fix Chapter Title drop shadows in Precal Trio

### ISSUE THREAD

**Issue by theresamasciale** [openstax/cnx-recipes#4086]
# Brief Description of Change and Screenshots
What kind of change is this? [ ] recipe [x] styles [ ] i don't know

Xanedu did a preflight review of College Algebra in preparation for the 2nd editions. They noted that the drop shadow behind the chapter title check on chapter opener pages is low resolution. This will need to be fixed for the upcoming 2nd editions (so like end of september/early october).

# Acceptance Criteria
Make the drop shadow high resolution or remove it.

# Content Location(s)
collection ID: 
Precal Trio: 
College Algebra - col11759
Precalculus - col11667
Algebra & Trig - col11758

server: production
chapter/section: all chapter openers
surrounding text:

**Comment by lwtchu** [comment]
I DON"T THINK this is possible. but see if we can beef up PPI for the drop shadow???

**Comment by ajluk** [comment]
@lwtchu I have made a little research today and it appears that it is possible to increase the resolution of a drop-shadow! :tada:: 
Here is how it looked zoomed before in Acrobat:
![image](https://user-images.githubusercontent.com/55203598/135684125-d72b9b39-b382-4470-8290-7630356f124b.png)
 and here is how it looks after:

**Comment by lwtchu** [comment]
WOW ! 300 dpi for the drop shadow is fine - we usually have transparent things (like transparent pngs) set at 300 so 300 for this is good. 

thank you <3
